Tools/MonoGame.Content.Builder/version/Version.props

```diff
@@ -0,0 +1,5 @@
+﻿<Project>
+  <PropertyGroup>
+    <MonoGameContentBuilderVersion>$([System.IO.Path]::GetFileName($([System.IO.Path]::GetDirectoryName($(MSBuildThisFileFullPath)))))</MonoGameContentBuilderVersion>
```

Comment:
```suggestion
    <MonoGameContentBuilderVersion>$([System.IO.Path]::GetFileName($(MSBuildThisFileDirectory)))</MonoGameContentBuilderVersion>
```

Reply:
Unfortunately this won't work; discussed in previous comment.